components/drawer-layout/DrawerMenuContent.tsx

```
@@ -23,7 +22,7 @@ import { useNavigation, useRouter } from "expo-router";
 import OrganizationSwitcherMenu from "../org-switcher";
 import { useOrganizationContext } from "@/contexts";
 
-interface DrawerMenuContentProps {}
+interface DrawerMenuContentProps { }
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)